openam-shared/src/main/java/org/forgerock/openam/utils/StringUtils.java

```diff
@@ -51,6 +58,22 @@ public static String insertContent(String original, int position, String content
         return original.substring(0, position) + content + original.substring(position);
     }
 
+    /**
+     * Compare two strings for equality.
+     * @param s1 first string to compare.
+     * @param s2 second string to compare.
+     * @return true if strings are equal or they are both null.
+     */
+    public static boolean isEqualTo(String s1, String s2) {
```

Comment:
probably just as unrelated

Reply:
I'll pop that into it's own commit

Reply:
Not required since pulling in OPENAM-4511